internal/cli/user/auth.go

```diff
@@ -40,8 +40,8 @@ func (creds Credentials) RedactedPrivateAPIKey() string {
 }
 
 // RedactedPassword returns the user's password with sensitive information redacted
-func (creds Credentials) RedactedPassword() string {
-	return redact(creds.Password)
+func RedactedPassword(password string) string {
+	return redact(password)
```

Comment:
[nit] this function isn't doing much anymore, we can probably just delete `RedactPassword` and export `Redact` to do the same thing... maybe we rename `RedactPrivateAPIKey` to something more general like `RedactKey`?